consensus/src/network_interface.rs

```diff
@@ -222,6 +222,7 @@ impl ApplicationNetworkSender<ConsensusMsg> for ConsensusNetworkSender {
         message: ConsensusMsg,
         timeout: Duration,
     ) -> Result<ConsensusMsg, RpcError> {
+        debug!("Sending an RPC {:?}", message);
```

Comment:
remove?

Reply:
well, how often do we do RPC? It might be useful to log them? otherwise I can remove